tools/linkage_disequilibrium/ld_tool

```diff
@@ -0,0 +1,401 @@
+#!/usr/bin/env perl
+# Copyright [2016-2017] EMBL-European Bioinformatics Institute
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+use strict;
+use warnings;
+
+use Getopt::Long;
+use Bio::EnsEMBL::Registry;
+use FileHandle;
+use List::Util qw(uniq);
+use Try::Tiny;
+
+my $DEFAULT_SPECIES = 'human';
+my $DEFAULT_HUMAN_ASSEMBLY = 'grch38';
+my $config = {};
+my $arg_count = scalar @ARGV;
+my @argv_copy = @ARGV;
+my @populations = ();
+my @regions = ();
+my @variants = ();
+GetOptions(
+  $config,
+  'help',                    # displays help message
+  'input_file|i=s',          # input file name
+  'output_file|o=s',         # default current directory date-time-calculation_type.out | inputfile_name.out
+  'warnings_file|w=s',       # write warnings to file
+  'species',                 # default human 
+  'population|p=s{1,}' => \@populations, # list all populations for LD computation: --populations population1 population2
+  'show_populations',            # show all available populations
+  'region=s{1,}' => \@regions,   # chromosome:start-end
+  'variant=s{1,}' => \@variants, # variant identifier  
+  'calculation|c=s',             # region, center, pairwise 
+  'assembly=s',                  # default human grch38
+  'r2=f',                        # value between 0 and 1
+  'd_prime=f',                   # value between 0 and 1
+  'add_variant_attributes',      # add variant attributes: variant_evidence, variant_consequence  
+) or die "ERROR: Failed to parse command-line flags\n";
+
+&usage && exit(0) if (!$arg_count) || $config->{help};
+$config->{species} ||= $DEFAULT_SPECIES;
+$config->{assembly} ||= $DEFAULT_HUMAN_ASSEMBLY;
+&init_connections($config);
+&show_populations($config) && exit(0) if ($config->{show_populations});
+
+die "Select at least one calculation: ./ld_tool --calculation [region|center|pairwise]. Or use --help for more information." unless ($config->{calculation});
```

Comment:
This needs a '\n' on the end of hte message else you also see ' at tools/linkage_disequilibrium/ld_tool line 57.'